scripts/style.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
```

Comment:
not sure. we though it would be nix. `nix run .#fmt`

Reply:
yes, I messed up in the rebase, I'll remove this